integration_test/ops_agent_test.go

```diff
@@ -2461,7 +2458,16 @@ func runResourceDetectorCli(ctx context.Context, logger *logging.DirectoryLogger
 			remote: "go.sum"},
 	}
 
+	// Create the folder structure on the VM
 	workDir := path.Join(workDirForPlatform(vm.Platform), "run_resource_detector")
+	packageDir := path.Join(workDir, "confgenerator", "resourcedetector")
+	createFolderCmd := fmt.Sprintf("mkdir -p %s", packageDir)
```

Comment:
Nit: since `runResourceDetectorCli` now runs on Windows targets then `mkdir -p s` isn't _technically_ correct, but I guess we got lucky since in PowerShell `mkdir %s` achieves the desired behaviour and the `-p` is ignored instead of producing an error. We should either provide an alternative command for Windows, or if not then at least add a comment here noting that `mkdir -p %s` happens to work on both platforms.

Reply:
This is good suggestion. I didn't know that PS would ignore `-p`. I agree I think it is better to have the alternative command for Windows here.